client/flutter/lib/home_page.dart

```diff
@@ -7,43 +9,56 @@ import 'package:WHOFlutter/pages/who_myth_busters.dart';
 import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
 import 'package:share/share.dart';
+import 'package:url_launcher/url_launcher.dart';
 
 class HomePage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
+    final localizations = S.of(context);
+
     return PageScaffold(
-        body: Column(
-      mainAxisSize: MainAxisSize.max,
-      mainAxisAlignment: MainAxisAlignment.center,
-      crossAxisAlignment: CrossAxisAlignment.stretch,
-      children: <Widget>[
-        Spacer(flex: 1),
-        PageButton(
-            title: S.of(context).protectYourself,
-            onPressed: () => Navigator.of(context)
-                .push(MaterialPageRoute(builder: (c) => ProtectYourself()))),
-        PageButton(
-            title: S.of(context).whoMythBusters,
-            onPressed: () => Navigator.of(context)
-                .push(MaterialPageRoute(builder: (c) => WhoMythBusters()))),
-        PageButton(
-            title: S.of(context).travelAdvice,
-            onPressed: () => Navigator.of(context)
-                .push(MaterialPageRoute(builder: (c) => TravelAdvice()))),
-        PageButton(
-            title: S.of(context).shareTheApp,
+      body: Column(
+        mainAxisSize: MainAxisSize.max,
+        mainAxisAlignment: MainAxisAlignment.center,
+        crossAxisAlignment: CrossAxisAlignment.stretch,
+        children: [
+          Spacer(flex: 1),
+          PageButton(
+            title: localizations.protectYourself,
+            onPressed: () => Navigator.of(context).push(MaterialPageRoute(builder: (c) => ProtectYourself())),
+          ),
+          PageButton(
+            title: localizations.whoMythBusters,
+            onPressed: () => Navigator.of(context).push(MaterialPageRoute(builder: (c) => WhoMythBusters())),
+          ),
+          PageButton(
+            title: localizations.travelAdvice,
+            onPressed: () => Navigator.of(context).push(MaterialPageRoute(builder: (c) => TravelAdvice())),
+          ),
+          PageButton(
+            title: localizations.shareTheApp,
             lightColor: true,
-            onPressed: () => Share.share(
-                'Check out the official COVID-19 Guide App https://www.who.int/covid-19-app')),
-        PageButton(
-            title: S.of(context).aboutTheApp,
+            onPressed: () async {
+              final url = 'https://www.who.int/covid-19-app';
+              if (Platform.isMacOS) {
```

Comment:
I'd prefer to instead catch the exception and then either show a message or open the URL.
That way we can handle it properly for all platforms that don't support sharing, this will also be more robust if we add platforms in the future, or if the `Share.share` adds support for MacOS.

Reply:
See #442